modules/standard/FileSystem.chpl

```diff
@@ -592,7 +592,20 @@ proc exists(name: string): bool throws {
 
    :yield:  The paths to any files found, relative to `startdir`, as strings
 */
+iter findFiles(startdir: string = ".", recursive: bool = false,
```

Comment:
Are we happy with the formal name `startdir`? Those are a huge PITA to change, though...

Reply:
There isn't an issue about the formal name, so either it didn't come up in the module review or people agreed to leave it. Looking at the module review notes I'm guessing it didn't come up.